packages/orchestrator-ui-components/src/components/WFOPageTemplate/WFOSidebar/WFOSidebar.tsx

```diff
@@ -17,11 +17,17 @@ import { WFOStartCreateWorkflowButtonComboBox } from './WFOStartCreateWorkflowBu
 
 export const WFOSidebar: FC = () => {
     const router = useRouter();
+    const [isSideNavOpenOnMobile, setisSideNavOpenOnMobile] = useState(false);
+
+    const toggleOpenOnMobile = () => {
+        setisSideNavOpenOnMobile(!isSideNavOpenOnMobile);
+    };
 
     return (
         <EuiSideNav
-            mobileTitle="Nav Items"
-            isOpenOnMobile={false}
+            mobileTitle="Main menu"
```

Comment:
Translate?

Reply:
Changed it: but now it looks even more distracting (the menu itself is all EN)